src/Domain/UseCase/Register.ts

```diff
@@ -57,6 +59,8 @@ export class Register implements UseCaseInterface {
 
     user = await this.userRepository.save(user)
 
+    await this.userKeyRotator.rotateServerKey(user)
```

Comment:
How are keys generated for existing users?

Reply:
see: https://github.com/standardnotes/auth/pull/4#issuecomment-789667579

Reply:
Not sure if it's a necessary optimization, but you can consider generating a user key only when it is needed. This way when you need to rotate server keys, you only have to do it for a smaller subset of users rather than all users.

But if at some point the server key will be used in many different areas then this may not be necessary.

Reply:
I was thinking about that but it seems kinda hacky to initialize the server key only when you don't have it and when saving a setting. 

Would make sense to create a async background job for rotating the keys of all users - defnitely something that we will also need in the future